### PR TITLE
doctl-databases-*: add page

### PR DESCRIPTION
**Pull request by kicka5h**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**
doctl v1.98.1

### CODE REVIEW

File: pages/common/doctl-databases-db.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# doctl databases db
+
+> Manage specific databases that are served by a database cluster.
+> More information: <https://docs.digitalocean.com/reference/doctl/reference/databases/db/>.
+
+- Run a doctl databases db command with an access token:
```
Comment (reinhart1010) [inline pages/common/doctl-databases-db.md:6]:
```suggestion
- Run a `doctl databases db` command with an access token:
```
Use backticks to clarify that these sre executable commands.

File: pages/common/doctl-databases-db.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+`doctl databases db [command] --access-token {{ access_token }}`
```
Comment (reinhart1010) [inline pages/common/doctl-databases-db.md:8]:
```suggestion
`doctl databases db [command] --access-token {{access_token}}`
```

No leading and trailing spaces on placeholder names, [like our style guide](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#placeholder-syntax).

File: pages/common/doctl-databases-db.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+- Retrieve the name of the given database hosted in the given database cluster:
+
+`doctl databases db get {{ database id }} {{ db name }}`
```
Comment (reinhart1010) [inline pages/common/doctl-databases-db.md:12]:
```suggestion
`doctl databases db get {{database_id}} {{db_name}}`
```
[Our style guide](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#placeholder-syntax) mandates that placeholders_are_written_in_snake_case.

File: pages/common/doctl-databases-db.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+- List existing databases hosted within a given database cluster:
+
+`doctl databases db list {{ database id }}`
```
Comment (reinhart1010) [inline pages/common/doctl-databases-db.md:16]:
```suggestion
`doctl databases db list {{database_id}}`
```

File: pages/common/doctl-databases-db.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+
+- Create a database with the given name in the given database cluster:
+
+`doctl databases db create {{ database id }} {{ db name }}`
```
Comment (reinhart1010) [inline pages/common/doctl-databases-db.md:20]:
```suggestion
`doctl databases db create {{database_id}} {{db_name}}`
```

File: pages/common/doctl-databases-db.md [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+
+- Delete the database with the given name in the given database cluster:
+
+`doctl databases db delete {{ database id }} {{ db name }}`
```
Comment (reinhart1010) [inline pages/common/doctl-databases-db.md:24]:
```suggestion
`doctl databases db delete {{database_id}} {{db_name}}`
```

**Review by reinhart1010 — CHANGES_REQUESTED** [review]
Thanks for contributing! Before I and other maintainers can approve, please fix the following writing style issues (and check out our [style guide](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#placeholder-syntax) for reference):

+ Use backticks (\`...\`, aka. Markdown inline code syntax) for highlighting code in example descriptions, such as "run a `doctl databases` command…" instead of "run a doctl databases command…".
+ When writing placeholders like `{{this}}`, **do not** add leading and trailing space to the placeholder names, like `{{ this }}`.
+ `{{long_variables_like_this_should_all_be_written_in_snake_case}}`.

I've written some examples for the `doctl databases db` subcommand; please revise the other pages too.

File: pages/common/doctl-databases-maintenance-window.md [hunk 7]
```diff
@@ -0,0 +1,16 @@
+# doctl databases maintenance-window
+
+> Schedule, and check the schedule of, maintenance windows for your databases.
+> More information: <https://docs.digitalocean.com/reference/doctl/reference/databases/maintenance-window/>.
+
+- Run a `doctl databases maintenance-window` command with an access token:
+
+`doctl databases maintenance-window [command] --access-token {{access_token}}`
```
Comment (reinhart1010) [inline pages/common/doctl-databases-maintenance-window.md:8]:
```suggestion
`doctl databases maintenance-window {{command}} --access-token {{access_token}}`
```

File: pages/common/doctl-databases-options.md [hunk 8]
```diff
@@ -0,0 +1,24 @@
+# doctl databases options
+
+> Enable the navigation of available options under each database engine.
+> More information: <https://docs.digitalocean.com/reference/doctl/reference/databases/options/>.
+
+- Run a `doctl databases options` command with an access token:
+
+`doctl databases options [command] --access-token {{access_token}}`
```
Comment (reinhart1010) [inline pages/common/doctl-databases-options.md:8]:
```suggestion
`doctl databases options {{command}} --access-token {{access_token}}`
```

File: pages/common/doctl-databases-pool.md [hunk 9]
```diff
@@ -0,0 +1,24 @@
+# doctl databases pool
+
+> Manage connection pools for your database cluster.
+> More information: <https://docs.digitalocean.com/reference/doctl/reference/databases/pool/>.
+
+- Run a `doctl databases pool` command with an access token:
+
+`doctl databases pool [command] --access-token {{access_token}}`
```
Comment (reinhart1010) [inline pages/common/doctl-databases-pool.md:8]:
```suggestion
`doctl databases pool {{command}} --access-token {{access_token}}`
```

File: pages/common/doctl-databases-replica.md [hunk 10]
```diff
@@ -0,0 +1,24 @@
+# doctl databases replica
+
+> Manage read-only replicas associated with a database cluster.
+> More information: <https://docs.digitalocean.com/reference/doctl/reference/databases/replica/>.
+
+- Run a `doctl databases replica` command with an access token:
+
+`doctl databases pool [command] --access-token {{access_token}}`
```
Comment (reinhart1010) [inline pages/common/doctl-databases-replica.md:8]:
```suggestion
`doctl databases pool {{command}} --access-token {{access_token}}`
```

File: pages/common/doctl-databases-sql-mode.md [hunk 11]
```diff
@@ -0,0 +1,16 @@
+# doctl databases sql-mode
+
+> View and configure a MySQL database cluster’s global SQL modes.
+> More information: <https://docs.digitalocean.com/reference/doctl/reference/databases/sql-mode/>.
+
+- Run a `doctl databases sql-mode` command with an access token:
+
+`doctl databases sql-mode [command] --access-token {{access_token}}`
```
Comment (reinhart1010) [inline pages/common/doctl-databases-sql-mode.md:8]:
```suggestion
`doctl databases sql-mode {{command}} --access-token {{access_token}}`
```

File: pages/common/doctl-databases-user.md [hunk 12]
```diff
@@ -0,0 +1,32 @@
+# doctl databases user
+
+> View details for, and create, database users.
+> More information: <https://docs.digitalocean.com/reference/doctl/reference/databases/user/>.
+
+- Run a `doctl databases user` command with an access token:
+
+`doctl databases user [command] --access-token {{access_token}}`
```
Comment (reinhart1010) [inline pages/common/doctl-databases-user.md:8]:
```suggestion
`doctl databases user {{command}} --access-token {{access_token}}`
```

File: pages/common/doctl-databases.md [hunk 13]
```diff
@@ -0,0 +1,24 @@
+# doctl databases
+
+> Manage your MySQL, Redis, PostgreSQL, and MongoDB database services.
+> More information: <https://docs.digitalocean.com/reference/doctl/reference/databases/>.
+
+- Run a `doctl databases` command with an access token:
+
+`doctl databases [command] --access-token {{access_token}}`
```
Comment (reinhart1010) [inline pages/common/doctl-databases.md:8]:
```suggestion
`doctl databases {{command}} --access-token {{access_token}}`
```

**Review by reinhart1010 — CHANGES_REQUESTED** [review]
Just another minor change and you're done.

**Review by reinhart1010 — APPROVED** [review]
Thanks! We'll need another contributor to approve and merge this PR.

File: pages/common/doctl-databases-db.md [hunk 14]
```diff
@@ -0,0 +1,24 @@
+# doctl databases db
+
+> Manage specific databases that are served by a database cluster.
+> More information: <https://docs.digitalocean.com/reference/doctl/reference/databases/db/>.
```
Comment (gutjuri) [inline pages/common/doctl-databases-db.md:4]:
```suggestion
> More information: <https://docs.digitalocean.com/reference/doctl/reference/databases/db>.
```

File: pages/common/doctl-databases-firewalls.md [hunk 15]
```diff
@@ -0,0 +1,20 @@
+# doctl databases firewalls
+
+> Manage firewalls for database clusters.
+> More information: <https://docs.digitalocean.com/reference/doctl/reference/databases/firewalls/>.
```
Comment (gutjuri) [inline pages/common/doctl-databases-firewalls.md:4]:
```suggestion
> More information: <https://docs.digitalocean.com/reference/doctl/reference/databases/firewalls>.
```

File: pages/common/doctl-databases-firewalls.md [hunk 16]
```diff
@@ -0,0 +1,20 @@
...
+
+`doctl databases firewalls {{command}} --access-token {{access_token}}`
+
+- Retrieve a list of firewall rules for a given database:
+
+`doctl databases firewalls list`
+
+- Add a database firewall rule to a given database:
+
+`doctl databases firewalls append {{database_id}} --rule {{type (droplet, k8s, ip_addr, tag, or app)}}:{{value}}`
```
Comment (gutjuri) [inline pages/common/doctl-databases-firewalls.md:16]:
```suggestion
`doctl databases firewalls append {{database_id}} --rule {{droplet|k8s|ip_addr|tag|app}}:{{value}}`
```

see https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#grouping-placeholders

File: pages/common/doctl-databases-maintenance-window.md [hunk 17]
```diff
@@ -0,0 +1,16 @@
+# doctl databases maintenance-window
+
+> Schedule, and check the schedule of, maintenance windows for your databases.
+> More information: <https://docs.digitalocean.com/reference/doctl/reference/databases/maintenance-window/>.
```
Comment (gutjuri) [inline pages/common/doctl-databases-maintenance-window.md:4]:
```suggestion
> More information: <https://docs.digitalocean.com/reference/doctl/reference/databases/maintenance-window>.
```

File: pages/common/doctl-databases-maintenance-window.md [hunk 18]
```diff
@@ -0,0 +1,16 @@
...
+
+`doctl databases maintenance-window {{command}} --access-token {{access_token}}`
+
+- Retrieve details about a database cluster's maintenance windows:
+
+`doctl databases maintenance-window get {{database_id}}`
+
+- Update the maintenance window for a database cluster:
+
+`doctl databases maintenance-window update {{database_id}} --day {{day_of_the_week (e.g. monday)}} --hour {{24_hour (e.g. '16:00')}}`
```
Comment (gutjuri) [inline pages/common/doctl-databases-maintenance-window.md:16]: not sure if we can have `e.g.` in placeholders. @kbdharun what is your opinion on this?

File: pages/common/doctl-databases-options.md [hunk 19]
```diff
@@ -0,0 +1,24 @@
+# doctl databases options
+
+> Enable the navigation of available options under each database engine.
+> More information: <https://docs.digitalocean.com/reference/doctl/reference/databases/options/>.
```
Comment (gutjuri) [inline pages/common/doctl-databases-options.md:4]:
```suggestion
> More information: <https://docs.digitalocean.com/reference/doctl/reference/databases/options>.
```

File: pages/common/doctl-databases-options.md [hunk 20]
```diff
@@ -0,0 +1,24 @@
...
+
+`doctl databases options {{command}} --access-token {{access_token}}`
+
+- Retrieve a list of the available database engines:
+
+`doctl databases options engines`
+
+- Retrieve a list of the available regions for a given database engine:
+
+`doctl databases options regions --engine {{database_engine (pg, mysql, redis, mongodb)}}`
```
Comment (gutjuri) [inline pages/common/doctl-databases-options.md:16]:
```suggestion
`doctl databases options regions --engine {{pg|mysql|redis|mongodb}}`
```

File: pages/common/doctl-databases-options.md [hunk 21]
```diff
@@ -0,0 +1,24 @@
...
+
+- Retrieve a list of the available slugs for a given database engine:
+
+`doctl databases options slugs --engine {{database_engine (pg, mysql, redis, mongodb)}}`
```
Comment (gutjuri) [inline pages/common/doctl-databases-options.md:20]:
```suggestion
`doctl databases options slugs --engine {{pg|mysql|redis|mongodb}}`
```

File: pages/common/doctl-databases-options.md [hunk 22]
```diff
@@ -0,0 +1,24 @@
...
+
+- Retrieve a list of the available versions for a given database engine:
+
+`doctl databases options versions --engine {{database_engine (pg, mysql, redis, mongodb)}}`
```
Comment (gutjuri) [inline pages/common/doctl-databases-options.md:24]:
```suggestion
`doctl databases options versions --engine {{pg|mysql|redis|mongodb}}`
```

File: pages/common/doctl-databases-pool.md [hunk 23]
```diff
@@ -0,0 +1,24 @@
+# doctl databases pool
+
+> Manage connection pools for your database cluster.
+> More information: <https://docs.digitalocean.com/reference/doctl/reference/databases/pool/>.
```
Comment (gutjuri) [inline pages/common/doctl-databases-pool.md:4]:
```suggestion
> More information: <https://docs.digitalocean.com/reference/doctl/reference/databases/pool>.
```

File: pages/common/doctl-databases-replica.md [hunk 24]
```diff
@@ -0,0 +1,24 @@
+# doctl databases replica
+
+> Manage read-only replicas associated with a database cluster.
+> More information: <https://docs.digitalocean.com/reference/doctl/reference/databases/replica/>.
```
Comment (gutjuri) [inline pages/common/doctl-databases-replica.md:4]:
```suggestion
> More information: <https://docs.digitalocean.com/reference/doctl/reference/databases/replica>.
```

File: pages/common/doctl-databases-sql-mode.md [hunk 25]
```diff
@@ -0,0 +1,16 @@
+# doctl databases sql-mode
+
+> View and configure a MySQL database cluster’s global SQL modes.
+> More information: <https://docs.digitalocean.com/reference/doctl/reference/databases/sql-mode/>.
```
Comment (gutjuri) [inline pages/common/doctl-databases-sql-mode.md:4]:
```suggestion
> More information: <https://docs.digitalocean.com/reference/doctl/reference/databases/sql-mode>.
```

File: pages/common/doctl-databases-sql-mode.md [hunk 26]
```diff
@@ -0,0 +1,16 @@
...
+
+`doctl databases sql-mode {{command}} --access-token {{access_token}}`
+
+- Get a MySQL database cluster's SQL modes:
+
+`doctl databases sql-mode get {{database_id}}`
+
+- Overwrite a MySQL database cluster's SQL modes to the specified modes:
+
+`doctl databases sql-mode set {{database_id}} {{sql_mode_1}} {{sql_mode_x}}`
```
Comment (gutjuri) [inline pages/common/doctl-databases-sql-mode.md:16]:
```suggestion
`doctl databases sql-mode set {{database_id}} {{sql_mode_1 sql_mode_2 ...}}`
```

File: pages/common/doctl-databases-user.md [hunk 27]
```diff
@@ -0,0 +1,32 @@
+# doctl databases user
+
+> View details for, and create, database users.
+> More information: <https://docs.digitalocean.com/reference/doctl/reference/databases/user/>.
```
Comment (gutjuri) [inline pages/common/doctl-databases-user.md:4]:
```suggestion
> More information: <https://docs.digitalocean.com/reference/doctl/reference/databases/user>.
```

File: pages/common/doctl-databases-user.md [hunk 28]
```diff
@@ -0,0 +1,32 @@
...
+
+`doctl databases user get {{database_id}} {{user_name}}`
+
+- Retrieve a list of database users for a given database:
+
+`doctl databases user list {{database_id}}`
+
+- Reset the auth password for a given user:
+
+`doctl databases user reset {{ database id} } {{user_name}}`
```
Comment (gutjuri) [inline pages/common/doctl-databases-user.md:20]:
```suggestion
`doctl databases user reset {{database id}} {{user_name}}`
```

File: pages/common/doctl-databases-user.md [hunk 29]
```diff
@@ -0,0 +1,32 @@
...
+
+- Reset the MySQL auth plugn for a given user:
+
+`doctl databases user reset {{database_id}} {{user_name}} {{new_auth_mode (cahing, sh2_password, mysql_native_password)}}`
```
Comment (gutjuri) [inline pages/common/doctl-databases-user.md:24]:
```suggestion
`doctl databases user reset {{database_id}} {{user_name}} {{caching_sha2_password|mysql_native_password}}`
```

File: pages/common/doctl-databases.md [hunk 30]
```diff
@@ -0,0 +1,24 @@
+# doctl databases
+
+> Manage your MySQL, Redis, PostgreSQL, and MongoDB database services.
+> More information: <https://docs.digitalocean.com/reference/doctl/reference/databases/>.
```
Comment (gutjuri) [inline pages/common/doctl-databases.md:4]:
```suggestion
> More information: <https://docs.digitalocean.com/reference/doctl/reference/databases>.
```

**Review by gutjuri — CHANGES_REQUESTED** [review]
Some more formatting errors and typos.

Thank you very much for your contribution!

File: pages/common/doctl-databases-pool.md (hunk 23 above)
Comment (kbdharun) [reply pages/common/doctl-databases-pool.md:4]: I am resolving this suggestion as the site resolves to the address with `/`. We only remove it in cases when the site doesn't redirect to a URL with `/`.

**Review by gutjuri — APPROVED** [review]

File: pages/common/doctl-databases-maintenance-window.md (hunk 18 above)
Comment (kbdharun) [reply pages/common/doctl-databases-maintenance-window.md:16]:
```suggestion
`doctl databases maintenance-window update {{database_id}} --day {{day_of_the_week}} --hour {{hour_in_24_hours_format}}`
```

File: pages/common/doctl-databases-db.md [hunk 31]
```diff
@@ -0,0 +1,24 @@
...
+
+`doctl databases db list {{database_id}}`
+
+- Create a database with the given name in the given database cluster:
+
+`doctl databases db create {{database_id}} {{db_name}}`
+
+- Delete the database with the given name in the given database cluster:
+
+`doctl databases db delete {{database_id}} {{db_name}}`
```
Comment (kbdharun) [inline pages/common/doctl-databases-db.md:24]:
```suggestion
`doctl databases db get {{database_id}} {{database_name}}`

- List existing databases hosted within a given database cluster:

`doctl databases db list {{database_id}}`

- Create a database with the given name in the given database cluster:

`doctl databases db create {{database_id}} {{database_name}}`

- Delete the database with the given name in the given database cluster:

`doctl databases db delete {{database_id}} {{database_name}}`
```

**Review by kbdharun — APPROVED** [review]
LGTM, Welcome to tldr and thanks for your contribution with this massive PR ❤️ .

**Pull request merged**